packages/blade/src/components/Collapsible/_decisions/decisions.md

```diff
@@ -0,0 +1,186 @@
+# Collapsible
+
+Collapsible is used to allow users to toggle the visibility of hidden content within a container.
+
+This document outlines the API of `Collapsible` component. This shares similarities with [Accordion's API](https://github.com/razorpay/blade/pull/1189).
+
+<div align="center">
+  <img src="./collapsible-thumbnail.png" width="380" alt="" />
+</div>
+
+## Design
+
+- [Figma - Collapsible](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=79-629874&t=sVxH3DOnx3L3F9rO-0)
+
+## Interactions
```

Comment:
Can we add animations that show a collapsible button/link instead of accordion? Seems a bit confusing

Reply:
Actually there is no figma prototype for this, we're reusing the same guidelines from accordion 😅

Reply:
Can you ask RK to create one because its okay to have redundant but contextual animations esp to avoid confusion